x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/upgrade_prebuilt_rules_table/use_upgrade_prebuilt_rules_table_columns.tsx

```diff
@@ -85,14 +85,20 @@ const INTEGRATIONS_COLUMN: TableColumn = {
 
 const createUpgradeButtonColumn = (
   upgradeOneRule: UpgradePrebuiltRulesTableActions['upgradeOneRule'],
-  loadingRules: RuleSignatureId[]
+  loadingRules: RuleSignatureId[],
+  isFetchingRulesPackage: boolean
```

Comment:
Same here. I think the 'install' button should not be aware of the rules package. Can we replace `isFetchingRulesPackage` with something more generic, such as `isDisabled`?